.github/workflows/ci.yml

```diff
@@ -40,4 +40,18 @@ jobs:
       - name: "Upload coverage to Codecov"
         uses: "codecov/codecov-action@v1"
         with:
-          fail_ci_if_error: true
\ No newline at end of file
+          fail_ci_if_error: true
+
+  flake8:
```

Comment:
well, I am not much fun of Tox as it is not very flexible (in my opinion) and do cache on env is somehow harder...
so if I read id correctly, when you run python 3.8 you run also all linters, so if the check fails there is no simple way to see if it failed because of the particular python version or other linters... correct?